Samples/ShopifyAcceleratedCheckoutsApp/ShopifyAcceleratedCheckoutsApp/Views/Components/ButtonSet.swift

```diff
@@ -62,11 +62,12 @@ struct ButtonSet: View {
                let productVariant = merchandise.asProductVariant
             {
                 CheckoutSection(title: "AcceleratedCheckoutButtons(variantID: quantity:)") {
-                    // Variant-based checkout with separate handlers
+                    // Variant-based checkout with separate handlers and custom corner radius
                     AcceleratedCheckoutButtons(
                         variantID: productVariant.id,
                         quantity: firstVariantQuantity
                     )
+                    .withCornerRadius(24)
```

Comment:
Heads up, the "with" prefixes [diverge from what we use currently for the `CheckoutSheet` usage](https://github.com/Shopify/checkout-sheet-kit-swift?tab=readme-ov-file#swiftui-usage) in Swift UI. I'd recommend `.cornerRadius(24)` instead.

As an aside; it appears `cornerRadius` is actually [deprecated in Swift now](https://developer.apple.com/documentation/swiftui/view/cornerradius(_:antialiased:)) and `.clipShape` is recommended instead 😕

Reply:
We are already using clipShape in this PR, not borderRadius.

`.clipShape(RoundedRectangle(cornerRadius: radius >= 0 ? radius : defaultCornerRadius))`

I'll update the naming.

Reply:
Renamed in https://github.com/Shopify/checkout-sheet-kit-swift/pull/331/commits/6db36fbd418fcd431fa776a33ea3c845fe4438e5